docs/figures/acts_logo_b_w.svg

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Since the logo is white-only the file name should be `acts_logo_white.svg` to avoid confusion.

Reply:
Sure, done.